sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -25,6 +25,11 @@ use crate::util::printer;
 
 /// Represents an OpenOCD server that we can interact with.
 pub struct OpenOcdServer {
+    /// Configuration file for the particular adapter used.  If given, will override the one from
```

Comment:
I don't know if overriding the command line one is the best thing to do as it could lead to counter-intuitive results? Other options would be to simply pass the jtag_cmsis_dap_adapter_cfg file on the command line for hyperdebug tests, or have an extra CLI argument for the hyperdebug adapter config and let the backend (cw/hyperdebug) choose which config to use?

Reply:
I would like to work towards a solution in which the `hyperdebug` backend driver knows and passes the file that fits it, and the existing `chip_whisperer` passes the file that fits it.  That is, the backend configuration file is not passed on the command line.

I did not want to possibly disrupt the existing driver, that is why I first introduce the override field, and then next, that could become the only way to specify the OpenOCD backend configuration.